src/main/java/org/apache/datasketches/quantiles/ItemsSketchIterator.java

```diff
@@ -19,35 +19,42 @@
 
 package org.apache.datasketches.quantiles;
 
+import java.util.Objects;
+
+import org.apache.datasketches.common.SketchesStateException;
 import org.apache.datasketches.quantilescommon.QuantilesGenericSketchIterator;
 
 /**
  * Iterator over ItemsSketch. The order is not defined.
  * @param <T> type of item
  */
 public class ItemsSketchIterator<T> implements QuantilesGenericSketchIterator<T> {
-
-  private final ItemsSketch<T> sketch;
+  //private final ItemsSketch<T> sketch;
```

Comment:
can remove this entirely if it's unneeded

Reply:
I removed this comment and will bundle this fix with changes to the HLL Javadocs.